packages/uikit-workshop/src/scripts/components/pl-nav/pl-nav.js

```diff
@@ -75,7 +75,7 @@ const SubSubList = props => {
         </Button>
       )}
 
-      {((viewAllItems.length > 0 && nonViewAllItems.length > 1) ||
+      {((viewAllItems.length > 0 && nonViewAllItems.length >= 1) ||
```

Comment:
`>0` is equivalent to `>= 1`, so you can just write this line: 

```
{((viewAllItems.length && nonViewAllItems.length) ||
```

Reply:
@illepic lol - you see, this is why I don't do the JavaScripts at 10pm on a work night 😛